src/Layer/FeatureGeometryLayer.js

```diff
@@ -0,0 +1,41 @@
+import { Group } from 'three';
+
+import GeometryLayer from 'Layer/GeometryLayer';
+import FeatureProcessing from 'Process/FeatureProcessing';
+import Feature2Mesh from 'Converter/Feature2Mesh';
+
+/**
+ * `FeatureGeometryLayer` is GeometryLayer.
```

Comment:
Maybe you could specify what is FeatureGeometryLayer compared to GeometryLayer ?